command/ca/revoke.go

```diff
@@ -0,0 +1,391 @@
+package ca
+
+import (
+	"crypto/tls"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/smallstep/certificates/api"
+	"github.com/smallstep/certificates/ca"
+	"github.com/smallstep/cli/command"
+	"github.com/smallstep/cli/crypto/pemutil"
+	"github.com/smallstep/cli/crypto/pki"
+	"github.com/smallstep/cli/crypto/x509util"
+	"github.com/smallstep/cli/errs"
+	"github.com/smallstep/cli/jose"
+	"github.com/smallstep/cli/ui"
+	"github.com/urfave/cli"
+	"golang.org/x/crypto/ocsp"
+)
+
+func revokeCertificateCommand() cli.Command {
+	return cli.Command{
+		Name:   "revoke",
+		Action: command.ActionFunc(revokeCertificateAction),
+		Usage:  "revoke a certificate",
+		UsageText: `**step ca revoke**
+[**--crt**=<certificate>] [**--key**=<key>] [**--token**=<ott>]
+[**-serial**=<serial-number>] [**--ca-url**=<uri>] [**--root**=<file>]
+[**--reason**=<reason>] [**--reasonCode**=<code>] [**-offline**]`,
```

Comment:
`--serial` is not currently being used and `<serial-number>` is missing as a positional argument.